modules/samples/producers/kafka/src/main/java/org/wso2/carbon/sample/kafka/Kafka.java

```diff
@@ -40,6 +41,7 @@
 
     public static void main(String args[]) {
 
+        log.info(Arrays.deepToString(args));
         System.out.println("Starting Kafka Client");
```

Comment:
Let's replace the sysout with proper logs. Also if you are logging the arguments, provide some sort of message to identify the log entry properly.